torch/_inductor/fx_passes/quantization.py

```diff
@@ -447,6 +447,20 @@ def qlinear(match: Match, *args, **kwargs):
     return qlinear
 
 
+def _reselect_binary_op(ori_binary_op_name, other, computation_node):
```

Comment:
nit: full name is better? e.g. `original_binary_op_name`?
Also suggest to add why we need to "reselect" in the code comment.

Reply:
Done, thanks for your comments!